proto/opamp.proto

```diff
@@ -817,3 +833,33 @@ message AgentConfigFile {
     // example "text/yaml".
     string content_type = 2;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////
+// Custom messages
+/////////////////////////////////////////////////////////////////////////////////////
+
+message CustomCapabilities {
+    // A list of custom capabilities that are supported. Each capability is a reverse FQDN
+    // with optional version information that uniquely identifies the custom capability
+    // and should match a capability specified in a supported CustomMessage.
+    // Status: [Development]
+    repeated string capabilities = 1;
+}
+
+message CustomMessage {
+    // A reverse FQDN that uniquely identifies the capability and matches one of the
+    // capabilities in the CustomCapabilities message.
+    // Status: [Development]
+    string capability = 1;
```

Comment:
So, if an interaction involves multiple capabilities it requires multiple AgentToServer/ServerToAgent messages, right?.

Reply:
A capability describes a custom feature that is supported and several custom messages may be involved in implementing that feature. For example, `io.opentelemetry.pause` which is described in the `Examples` section uses `pause`, `status`, and `error` messages. The message types are specific to the capability and the number of messages required and the direction of the messages is defined by the capability.